File: Acute_Inflammation1/predict.m
Min = 35;
Max = 42;
li = zeros(1,7);
li(1) = 1;
li(2) = input("\nTemperature of patient { 35C-42C }: ");
li(2) = ( li(2) - Min )/( Max - Min );
li(3) = input("\nOccurrence of nausea { 0, 1}: ");
li(4) = input("\nLumbar pain { 0, 1 }: ");
li(5) = input("\nUrine pushing (continuous need for urination) { 1, 0 }: ");
li(6) = input("\nMicturition pains { 1, 0 }: ");
li(7) = input("\nBurning of urethra, itch, swelling of urethra outlet { 1, 0 }: ");

theta = [-6.40242 ,-2.53137 ,6.41191 ,-10.74731 ,13.87736 ,6.53072 ,-1.73427; -19.76393 ,7.38195 ,10.18248 ,10.87613 ,5.57733 ,-0.60511 ,5.65223];
x = li*theta';

if( x(1) >= 0.5 ),
	display("\nInflammation of urinary bladder: Yes\n");
else
	display("\nInflammation of urinary bladder: No\n");
endif;

if( x(2) >= 0.5 ),
	display("Nephritis of renal pelvis origin: Yes\n");
else
	display("Nephritis of renal pelvis origin: No\n");
endif;

File: Acute_Inflammation/predict.py
li = []*7
li[0] = 1
print("Temperature of patient { 35C-42C }")
li[1] = (input()-35)/(42-35)
print("Occurrence of nausea { 0, 1}")
li[2] = input()
print("Lumbar pain { 0, 1 }")
li[3] = input()
print("Urine pushing (continuous need for urination) { 1, 0 }")
li[4] = input()
print("Micturition pains { 1, 0 }")
li[5] = input()
print("Burning of urethra, itch, swelling of urethra outlet { 1, 0 }")
li[6] = input()

theta = [[-4.40780,-5.22960,5.06284,-9.83564,12.68278,7.42017,-1.52519],
   		[-5.39841,-2.74480,-0.37658,-2.51075,-2.30247,-1.26762,-1.07094],
    	[4.93700,5.55125,-5.92033,11.04599,-13.64938,-8.77261,2.17064]]
